coalib/misc/Shell.py

```diff
@@ -52,6 +54,13 @@ def run_interactive_shell_command(command, **kwargs):
     """
     if not kwargs.get('shell', False) and isinstance(command, str):
         command = shlex.split(command)
+    else:
+        command = list(command)
+
+    if platform.system() == 'Windows':  # pragma: no cover
```

Comment:
please do not use `# pragma: no cover` for OS-specific code.  At least use `# pragma (posix|nt): no cover` .